### PR TITLE
Begin 19w34a mappings

### PR DESCRIPTION
**Pull request by Shnupbups**
I will continue to add onto this

### CODE REVIEW

File: mappings/net/minecraft/client/render/entity/EntityRenderer.mapping [hunk 1]
```diff
@@ -39,7 +39,7 @@ CLASS net/minecraft/class_897 net/minecraft/client/render/entity/EntityRenderer
 		ARG 6 z
 		ARG 8 text
 	METHOD method_3931 getTexture (Lnet/minecraft/class_1297;)Lnet/minecraft/class_2960;
-		ARG 1 boat
+		ARG 1 bee
```
Comment (liach) [inline mappings/net/minecraft/client/render/entity/EntityRenderer.mapping]: seriously?

File: mappings/net/minecraft/entity/passive/BeeEntity.mapping [hunk 2]
```diff
@@ -0,0 +1,29 @@
+CLASS net/minecraft/class_4466 net/minecraft/entity/passive/BeeEntity
+	CLASS class_4467
+		METHOD method_21814 canEnterHive ()Z
+	CLASS class_4468 BeeStingGoal
```
Comment (liach) [inline mappings/net/minecraft/entity/passive/BeeEntity.mapping]: `StingGoal`

**Review by liach — COMMENTED** [review]
Keep mapping.

File: mappings/net/minecraft/client/render/entity/EntityRenderer.mapping (hunk 1 above)
Comment (Shnupbups) [reply mappings/net/minecraft/client/render/entity/EntityRenderer.mapping]: Hey, I didn't do that! Enigma did that for some reason.

Comment (liach) [reply mappings/net/minecraft/client/render/entity/EntityRenderer.mapping]: :trollface:

Comment (Sollace) [reply mappings/net/minecraft/client/render/entity/EntityRenderer.mapping]: Why was it called `boat` in the first place anyway? Shouldn't it be `entity`?

Comment (Shnupbups) [reply mappings/net/minecraft/client/render/entity/EntityRenderer.mapping]: Fixed!

**Comment by Shnupbups** [comment]
I'll keep mapping tomorrow, it's almost 3:30AM

**Review by modmuss50 — APPROVED** [review]
Looks good.

File: mappings/net/minecraft/block/entity/BeeHiveBlockEntity.mapping [hunk 3]
```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/class_4482 net/minecraft/block/entity/BeeHiveBlockEntity
+	CLASS class_4483
+		FIELD field_20425 entityData Lnet/minecraft/class_2487;
+	FIELD field_20423 bees Ljava/util/Map;
+	FIELD field_20424 flowerPos Lnet/minecraft/class_2338;
```
Comment (Runemoro) [inline mappings/net/minecraft/block/entity/BeeHiveBlockEntity.mapping]: What flower is this? `nearestFlowerPosition`?

File: mappings/net/minecraft/block/entity/BeeHiveBlockEntity.mapping [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+	METHOD method_21848 tryEnterHive (Lnet/minecraft/class_1297;Z)V
+	METHOD method_21849 tryEnterHive (Lnet/minecraft/class_1297;ZI)V
+	METHOD method_21856 isFullOfBees ()Z
+	METHOD method_21857 isOriginNotFlower ()Z
```
Comment (Runemoro) [inline mappings/net/minecraft/block/entity/BeeHiveBlockEntity.mapping]: What's the origin?

File: mappings/net/minecraft/entity/passive/BeeEntity.mapping [hunk 5]
```diff
@@ -0,0 +1,105 @@
+CLASS net/minecraft/class_4466 net/minecraft/entity/passive/BeeEntity
+	CLASS class_4467 NotAngryGoal
```
Comment (Runemoro) [inline mappings/net/minecraft/entity/passive/BeeEntity.mapping:2]: Aren't all goals except the sting goal "not angry" goals? What makes this one different?

File: mappings/net/minecraft/entity/passive/BeeEntity.mapping [hunk 6]
```diff
@@ -1,44 +1,105 @@
...
 	METHOD method_21806 setHasStung (Z)V
+		ARG 1 hasStung
 	METHOD method_21807 setCannotEnterHiveTicks (I)V
 		ARG 1 ticks
+	METHOD method_21808 setNearTarget (Z)V
+		ARG 1 isNearTarget
 	METHOD method_21809 setAnger (I)V
+		ARG 1 ticks
+	METHOD method_21810 setPollinating (Z)V
+		ARG 1 isPollinating
```
Comment (liach) [inline mappings/net/minecraft/entity/passive/BeeEntity.mapping]: `pollinating`

File: mappings/net/minecraft/entity/passive/BeeEntity.mapping [hunk 7]
```diff
@@ -1,44 +1,105 @@
...
+	METHOD method_21811 getBodyPitch (F)F
+		ARG 1 tickDelta
+	METHOD method_21812 getTrackerByte (I)Z
```
Comment (liach) [inline mappings/net/minecraft/entity/passive/BeeEntity.mapping]: `getFlag`

File: mappings/net/minecraft/entity/passive/BeeEntity.mapping [hunk 8]
```diff
@@ -1,44 +1,105 @@
...
+		ARG 1 location
+	METHOD method_21813 findPointOfInterest (I)Ljava/util/Optional;
```
Comment (liach) [inline mappings/net/minecraft/entity/passive/BeeEntity.mapping:104]: Is this the villager point of interest?

File: mappings/net/minecraft/entity/passive/BeeEntity.mapping [hunk 9]
```diff
@@ -1,44 +1,105 @@
...
+	METHOD method_21797 setFlowerPos (Lnet/minecraft/class_2338;)V
 		ARG 1 pos
 	METHOD method_21805 setHasNectar (Z)V
+		ARG 1 hasNectar
 	METHOD method_21806 setHasStung (Z)V
+		ARG 1 hasStung
 	METHOD method_21807 setCannotEnterHiveTicks (I)V
 		ARG 1 ticks
+	METHOD method_21808 setNearTarget (Z)V
+		ARG 1 isNearTarget
```
Comment (liach) [inline mappings/net/minecraft/entity/passive/BeeEntity.mapping]: `nearTarget`

File: mappings/net/minecraft/entity/passive/BeeEntity.mapping (hunk 7 above)
Comment (Shnupbups) [reply mappings/net/minecraft/entity/passive/BeeEntity.mapping]: `getFlag` already exists in `Entity`.

File: mappings/net/minecraft/entity/passive/BeeEntity.mapping (hunk 5 above)
Comment (Shnupbups) [reply mappings/net/minecraft/entity/passive/BeeEntity.mapping:2]: `NotAngryGoal` is abstract, and is extended by `EnterHiveGoal`, `GrowCropsGoal`, `FIndHiveGoal`, `PollinateGoal`, and `MoveToTargetGoal` (and by extension, `MoveToHiveGoal` and `MoveToFlowerGoal`).

`NotAngryGoal` only exists to make sure that those goals are only run when the bee is not angry. Therefore it makes sense for it to be `NotAngryGoal`.

File: mappings/net/minecraft/entity/passive/BeeEntity.mapping (hunk 7 above)
Comment (Shnupbups) [reply mappings/net/minecraft/entity/passive/BeeEntity.mapping]: I'll make it `getBeeFlag` (and `setBeeFlag` to match)

File: mappings/net/minecraft/entity/passive/BeeEntity.mapping (hunk 8 above)
Comment (Shnupbups) [reply mappings/net/minecraft/entity/passive/BeeEntity.mapping:104]: Yes, Bees use the same system to find Hives and Nests.

File: mappings/net/minecraft/block/entity/BeeHiveBlockEntity.mapping (hunk 4 above)
Comment (Shnupbups) [reply mappings/net/minecraft/block/entity/BeeHiveBlockEntity.mapping]: The world origin, (0,0,0). The hive sets its `flowerPos` to that by default. I've changed it to `hasFlowerPos`.

**Review by Runemoro — APPROVED** [review]

**Pull request merged**